### DISCUSSION TITLE
location of v2ray log files in x-ui

### DISCUSSION THREAD

**Discussion by farzinsb** [FranzKafkaYu/x-ui#120] [General]
hi, where is the log files of v2ray are being stored in the server when we install it using x-ui script?
there is no files in /var/log/v2ray

**Comment by farzinsb** [comment]
I checked the config , it already had that logger service line.
how should I check if its enabled (its working) or not ?

**Reply by FranzKafkaYu** [reply]
> I checked the config , it already had that logger service line. how should I check if its enabled (its working) or not ?

can you show me your config?

**Reply by farzinsb** [reply]
> 

```
{
  "api": {
    "services": [
      "HandlerService",
      "LoggerService",
      "StatsService"
    ],
    "tag": "api"
  },
  "inbounds": [
    {
      "listen": "127.0.0.1",
      "port": 62789,
      "protocol": "dokodemo-door",
      "settings": {
        "address": "127.0.0.1"
      },
      "tag": "api"
    }
  ],
  "outbounds": [
    {
      "protocol": "freedom",
      "settings": {}
    },
    {
      "protocol": "blackhole",
      "settings": {},
      "tag": "blocked"
    }
  ],
  "policy": {
    "system": {
      "statsInboundDownlink": true,
      "statsInboundUplink": true
    }
  },
  "routing": {
    "rules": [
      {
        "inboundTag": [
          "api"
        ],
        "outboundTag": "api",
        "type": "field"
      },
      {
        "ip": [
          "geoip:private"
        ],
        "outboundTag": "blocked",
        "type": "field"
      },
      {
        "outboundTag": "blocked",
        "protocol": [
          "bittorrent"
        ],
        "type": "field"
      }
    ]
  },
  "stats": {}
}
```

hi, here is the config. which is the default config.

**Reply by FranzKafkaYu** [reply]
> > 
> 
> ```
> {
>   "api": {
>     "services": [
>       "HandlerService",
>       "LoggerService",
>       "StatsService"
>     ],
>     "tag": "api"
>   },
>   "inbounds": [
>     {
>       "listen": "127.0.0.1",
>       "port": 62789,
>       "protocol": "dokodemo-door",
>       "settings": {
>         "address": "127.0.0.1"
>       },
>       "tag": "api"
>     }
>   ],
>   "outbounds": [
>     {
>       "protocol": "freedom",
>       "settings": {}
>     },
>     {
>       "protocol": "blackhole",
>       "settings": {},
>       "tag": "blocked"
>     }
>   ],
>   "policy": {
>     "system": {
>       "statsInboundDownlink": true,
>       "statsInboundUplink": true
>     }
>   },
>   "routing": {
>     "rules": [
>       {
>         "inboundTag": [
>           "api"
>         ],
>         "outboundTag": "api",
>         "type": "field"
>       },
>       {
>         "ip": [
>           "geoip:private"
>         ],
>         "outboundTag": "blocked",
>         "type": "field"
>       },
>       {
>         "outboundTag": "blocked",
>         "protocol": [
>           "bittorrent"
>         ],
>         "type": "field"
>       }
>     ]
>   },
>   "stats": {}
> }
> ```
> 
> hi, here is the config. which is the default config.

you didn't set log option here, please read WIKI

**Reply by farzinsb** [reply]
` "log": {
    "access": "/usr/local/x-ui/bin/access.log",
    "error": "/usr/local/x-ui/bin/error.log",
    "loglevel": "warning",
    "dnsLog": false
 },`

ok I added these lines to the config and it started writing them.  thanks.
